lightning/src/ln/async_payments_tests.rs

```diff
@@ -20,7 +20,7 @@ use crate::ln::msgs::{
 	BaseMessageHandler, ChannelMessageHandler, MessageSendEvent, OnionMessageHandler,
 };
 use crate::ln::offers_tests;
-use crate::ln::onion_utils::INVALID_ONION_BLINDING;
+use crate::ln::onion_utils::LocalHTLCFailureReason;
```

Comment:
This commit is just so intense. Getting nervous of reviewing so many trivial changes. I wonder if it is worth to:
* Do the renames from err_code to reason separately in a non-review commit
* Introduce the LocalHTLCFailureReason enum, still unused
* Have a commit that contains just mechanical search/replaces that needs no review. Possibly list all the replaces the were applied.
* Then finish with a commit with the semi-manual leftovers.

Maybe it's too much of an ask though.

Reply:
I'll take a shot at splitting this up 👍

Reply:
It is incredible how much more reviewable this change makes your PR. Thanks so much! The anxiety is gone 😅